### PR TITLE
Update about us to new committee

### PR DESCRIPTION
**Pull request by JMAlego**
This pull request adds the new committee to the website. I'm looking for feedback on whether people think I approached the jointly held positions correctly here as I added the "position jointly held" in the same style as the "currently on year in industry" text, but I'm not sure if that's the right approach.

### CODE REVIEW

File: regular/about.html [hunk 1]
```diff
@@ -43,29 +43,31 @@ <h2>Committee</h2>
 
 <dl>
   <dt>Chair</dt>
-  <dd>Jacob Allen, fourth year computer science undergraduate</dd>
+  <dd>Charlotte Wringe, third year computer science undergraduate</dd>
 
   <dt>Secretary</dt>
```
Comment (sersorrel) [inline regular/about.html]:
```suggestion
  <dt>Secretary (position jointly held)</dt>
```

File: regular/about.html [hunk 2]
```diff
@@ -43,29 +43,31 @@ <h2>Committee</h2>
...
-  <dd>Adam Birtles, third year computer science undergraduate</dd>
+  <dd>Adam Birtles, third year computer science undergraduate, position jointly held</dd>
+  <dd>Ash Holland, second year computer science undergraduate, position jointly held</dd>
```
Comment (sersorrel) [inline regular/about.html]:
```suggestion
  <dd>Adam Birtles, third year computer science undergraduate</dd>
  <dd>Ash Holland, second year computer science undergraduate</dd>
```

File: regular/about.html [hunk 3]
```diff
@@ -43,29 +43,31 @@ <h2>Committee</h2>
...
+  <dd>Daniel O’Brien, first year computer science undergraduate</dd>
 
   <dt>Press and Publicity</dt>
   <dd>Aaron Christiansen, second year computer science undergraduate</dd>
 
   <dt>Academic Events Officer</dt>
-  <dd>Charlotte Wringe, third year computer science undergraduate</dd>
+  <dd>Tara Harley, first year computer science undergraduate</dd>
   
   <dt>Infrastructure Officer</dt>
```
Comment (sersorrel) [inline regular/about.html]:
```suggestion
  <dt>Infrastructure Officer (position jointly held)</dt>
```

File: regular/about.html [hunk 4]
```diff
@@ -43,29 +43,31 @@ <h2>Committee</h2>
...
-  <dd>Luke Moll, third year computer science undergraduate</dd>
+  <dd>Luke Moll, third year computer science undergraduate, position jointly held</dd>
+  <dd>Ezekiel Bethel, second year physics undergraduate, position jointly held</dd>
```
Comment (sersorrel) [inline regular/about.html]:
```suggestion
  <dd>Luke Moll, third year computer science undergraduate</dd>
  <dd>Ezekiel Bethel, second year physics undergraduate</dd>
```

**Comment by JMAlego** [comment]
I've applied the changes from committee, if you're happy @sersorrel feel free to merge.

**Pull request merged**